back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/project_phase_started.rb

```diff
@@ -81,5 +83,13 @@ def generate_commands(recipient:, activity:, time: nil)
         delay: 8.hours.to_i
       }]
     end
+
+    private
+
+    def campaign_enabled?(activity:, time: nil)
+      return false if self.class.first.enabled == false
```

Comment:
```suggestion
      return false unless enabled
```
Would it be the same?

Reply:
Also, we already have this check in https://github.com/CitizenLabDotCo/citizenlab/blob/e8ddddab90b3de11bdaebd08921317f8e42c69ac/back/engines/free/email_campaigns/app/models/concerns/email_campaigns/disableable.rb#L22-L24

So, maybe we can just remove this line and rename this method to sth like `campaign_enabled_for_phase?`.

Reply:
Not sure I understand. If I remove this in `Disableable` then setting `enabled` will have no effect in any disableable campaign.

Reply:
Ah, OK, I think you mean the line in `project_phase_started.rb`

Reply:
Yes, I meant this line `return false if self.class.first.enabled == false`. Sorry for the confusion.